notebooks/official/automl/sdk_automl_text_sentiment_analysis_online.ipynb

```diff
@@ -29,7 +29,7 @@
         "id": "title"
```

Comment:
<p>move this pip into the above cell and drop the corresponding text cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1208/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>